common/changes/@itwin/grouping-mapping-widget/gm-context-uiProvider_2022-09-21-20-18.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/grouping-mapping-widget",
+      "comment": "add context custom UI support ",
```

Comment:
Change log comments should start with a capital letter. Also, drop the trailing space.